pulp_service/pulp_service/app/viewsets.py

```diff
@@ -181,3 +184,28 @@ def create(self, request):
 
         task = dispatch(dispatch_task, shared_resources=shared_resources, kwargs=kwargs)
         return OperationPostponedResponse(task, request)
+
+
+class TaskIngestionDispatcherView(APIView):
+
+    authentication_classes = []
+    permission_classes = []
+
+
+    def get(self, request=None, path=None, task_num=1):
```

Comment:
task_num should have no default. If unset it should loop forever until task_timeout time.